tfx/components/example_gen/import_example_gen/executor_test.py

```diff
@@ -69,6 +69,24 @@ def check_result(got):
 
       util.assert_that(examples, check_result)
 
+  def testImportSequenceExample(self):
```

Comment:
we feel don't need this test, as it's mainly for testing _ImportSerializedRecord (which doesn't contain a branch in it)